go/consensus/tendermint/apps/scheduler/scheduler.go

```diff
@@ -407,7 +409,7 @@ func (app *schedulerApplication) electCommittee(
 		return nil
 	}
 
-	nrNodes, wantedNodes := len(nodeList), workerSize+backupSize
+	nrNodes, wantedNodes := len(nodeList), minSize
```

Comment:
That's not what's meant by `MinPoolSize`. There should be an additional check above that fails in case `nrNodes < minSize` (e.g. there should be at least `MinPoolSize` eligible nodes).